LogsHandler/src/test/java/bootTest/IntegrationTest.java

```diff
@@ -42,32 +44,45 @@ public static void beforeAllTests() {
     public void testEndToEnd() {
         String key = RandomStringUtils.random(15, false, true);
         String jsonObjectAsString = "{\"message\":\"" + key + "\"}";
+        String account = "{'accountName':'fadi'}";
 
-        Response postResponse = webTarget.path("/index")
+        //create account
+        Response postResponse = webTarget.path("/create-account")
                 .request(MediaType.APPLICATION_JSON)
                 .header(HttpHeaders.USER_AGENT, userAgent)
-                .post(Entity.json(jsonObjectAsString));
-        assertEquals(200, postResponse.getStatus());
+                .header(HttpHeaders.CONTENT_TYPE, "application/json")
+                .post(Entity.json(account));
+        assertEquals(HttpURLConnection.HTTP_CREATED, postResponse.getStatus());
 
-        String result = postResponse.toString() + "\n" + postResponse.readEntity(String.class);
-        logger.debug(result);
+        String accountJsonString = postResponse.readEntity(String.class);
+        JsonObject accountJson = InfraUtil.stringToJson(accountJsonString);
```

Comment:
We prefer not to use `JsonObject` - let's use a concrete instance (POJO) instead.
In this case:
```suggestion
        Account accountJson = InfraUtil.stringToJson(accountJsonString, Account.class);
```